### DISCUSSION TITLE
Need to refresh the page twice to clear the session after deleting account.

### DISCUSSION THREAD

**Discussion by p1ckle-rick** [clerk/javascript#3368] [Support request - DO NOT OPEN HERE]
After I delete the account through the `UserButton` component I need to refresh the page twice for the session to be cleared. This causes the `SignInButton` to only show up after the second refresh.

I am following this guide: https://clerk.com/docs/quickstarts/nextjs

**Comment by BigMable007** [comment]
I am facing this issue as well.

**Reply by p1ckle-rick** [reply]
@BigMable007 I reported this on their Discord channel as well, I think they are still working on the fix.
